docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance/PFIHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Learners;
+using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic.PermutationFeatureImportance
+{
+    public class PfiHelper
```

Comment:
So you add BinaryClassificationExample and Regression to our documentation.
Is it ok to have reference to helper class in documentation?
I doubt content of this file would be visible to user.